wahoomc/timings.py

```diff
@@ -0,0 +1,46 @@
+"""
+functions for timing sections of code, and output results in log lines
+"""
+#!/usr/bin/python
+
+# import official python packages
+import time
+import logging
+
+log = logging.getLogger('main-logger')
+
+
+class Timings:
+    """Class for starting the wall time, and logging when done"""
+    wall_time_start = None
+    wall_time_end = None
+
+    def start_wall_time(self):
```

Comment:
```suggestion
    def __init__(self):
```

like you wrote by yourself :-) https://github.com/treee111/wahooMapsCreator/pull/225#discussion_r1382543030
plus removing all the calls of this method throughout the files